scripts/lochness_check_config.py

```diff
@@ -44,7 +45,8 @@ def check_lochness_configurations():
         check_keyring(keyring_dict)
 
     if args.keyring:
-        with open(args.keyring, 'r') as f:
+        keyring_path = Path(args.keyring)
```

Comment:
Why is `Path()` necessary?

Reply:
`Veracode` recommended avoiding direct use of `args` attributes.